### ISSUE TITLE
IIIF and rights metadata for digitized Voyager objects

### ISSUE THREAD

**Issue by flapka** [ycba-cia/blacklight-collections2#468]
This thread references multiple issues:

1. New YUL procedures for connecting analog objects to their digital representations, which YCBA will probably want to follow
2. Revised procedures for recording related rights information in MARC (carrying over discussion at the end of issue #382 )
3. And a dependency: mapping MARC holdings data to Blacklight (see issue #412 ).

The new YUL procedures, now in testing, add three 856 (linking) fields to every MARC *holding* for digitized objects. Example:

```
856 41 $u https://collections.library.yale.edu/catalog/12301559 $y Digital version
856 41 $a IIIF manifest $x https://collections.library.yale.edu/manifests/12301559
856 41 $a IIIF thumbnail $x https://collections.library.yale.edu/iiif/2/12297084/full/!300,300/0/default.jpg 
```

Full spec attached (Word file)
[Voyager + DCS 856 Linking Specification_August 2023 -- FL copy.docx](https://github.com/ycba-cia/blacklight-collections2/files/12832263/Voyager.%2B.DCS.856.Linking.Specification_August.2023.--.FL.copy.docx)

If we follow suit at YCBA, an example might look like this:
```
856 41 $u https://collections.britishart.yale.edu/catalog/orbis:3429277 $y Digital version
856 41 $a IIIF manifest $x https://manifests.collections.yale.edu/ycba/orb/3429277
856 41 $a IIIF thumbnail $x [pattern for thumbnail?]
```
The above would have no immediate impact in YCBA's Blacklight, but it would hopefully allow our images to be displayed in LUX and (eventually, perhaps) in Quicksearch, as soon as YUL makes the necessary changes to mappings etc.

In issue #382 we also talked about changing fields for rights information, using 856 instead of 542. If we take that course, I think the first of the three 856 fields might look like this:

`856 41 $u https://collections.britishart.yale.edu/catalog/orbis:3429277 $y Digital version $r http://rightsstatements.org/vocab/UND/1.0/ $t Copyright Undetermined `

If Blacklight needs access to that data, then of course we also need to be able to map/index Holdings data to BL, which still proves challenging (currently waiting for more input from Youn Noh on the possibilities of POD (described in #412 )).

YUL will use an automated process to add the 856 fields in Voyager. I don't know if the same is possible on our end.

**Comment by yulgit1** [comment]
Regarding rights and looking at the source MARC:

https://libapp.library.yale.edu/OAI_BAC/src/OAIOrbisTool.jsp?verb=GetRecord&identifier=oai:orbis.library.yale.edu:3429277&metadataPrefix=marc21

```
<marc:datafield tag="856" ind1="4" ind2="1">
<marc:subfield code="y">View a selection of digital images in the Yale Center for British Art's online catalogue</marc:subfield>
<marc:subfield code="u">https://collections.britishart.yale.edu/catalog/orbis:3429277</marc:subfield>
</marc:datafield>
```
Is it that the new 856 with $r and $t, not be here but in a separate holdings? IE:

https://libapp.library.yale.edu/VoySearch/GetAllMfhdItem
not
https://libapp.library.yale.edu/OAI_BAC/src/OAIOrbisTool.jsp

**Comment by flapka** [comment]
That's right, the new 856 (if following the new YUL procedure) would live in the MFHD (aka Holdings)
